README.md

```diff
@@ -1,2 +1,2 @@
-# Piwik Analytics Cloud by InnoCraft - Cloudflare App
+# Matomo Analytics Cloud by InnoCraft - Cloudflare App
```

Comment:
In the `package.json` "InnoCraft" has been replaced with "Matomo Team". Would it make sense to align the README then?

Reply:
Makes sense.
@mattab, what do you say?

Reply:
That's fine to change @Findus23

Reply:
done